src/commands/git-metadata/upload.ts

```diff
@@ -216,7 +217,7 @@ export class UploadCommand extends Command {
   private getApiRequestBuilder(apiKey: string): RequestBuilder {
     return getRequestBuilder({
       apiKey,
-      baseUrl: 'https://' + apiHost,
+      baseUrl: `https://api.${getDatadogSite()}`,
```

Comment:
nit: Small suggestion for consistency
```suggestion
      baseUrl: `https://api.${this.config.datadogSite}`,
```

Could you also update the line 225 below to use `this.config.datadogSite` instead ? 🙏